plugin/src/test/kotlin/org/jetbrains/intellij/utils/KotlinCode.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.intellij.utils
```

Comment:
This file looks quite strange. First two functions definitely should be inlined, others can be kept but with some better name.